recipes/keyring-pass/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "keyring-pass" %}
+{% set version = "0.9.3" %}
+{% set python_min = "3.7" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default. 

```suggestion
```